app/helpers/import_rooms.rb

```diff
@@ -0,0 +1,30 @@
+require 'csv'
+require 'point'
+require 'room'
+require 'building'
+require 'polyline'
+
+rooms = %w[A B C D E F G H I]
+building = Building.create
+
+csv_text = File.read('app/assets/data/importBuilding1.csv')
+csv = CSV.parse(csv_text)
+points = {}
+rooms.each do |name|
+  polygon = {}
+  csv.each do |row|
```

Comment:
Maybe a high-level comment like `"Collect all points of this room"` would be useful

Reply:
gonna add that